### DISCUSSION TITLE
What is the  fourth parameter of app.getRequestHandler() ?

### DISCUSSION THREAD

**Discussion by naseef0** [vercel/next.js#15314] [Help] [answered]
![Screenshot 2020-07-20 at 1 18 14 PM](https://user-images.githubusercontent.com/45784156/87912708-79be4900-ca8b-11ea-8151-9c9a827fc473.png)

![Screenshot 2020-07-20 at 1 18 41 PM](https://user-images.githubusercontent.com/45784156/87912746-8a6ebf00-ca8b-11ea-9126-f0903840b39d.png)

**Comment by c0b41** [comment, marked as answer]
there is no fourth paramater https://github.com/vercel/next.js/blob/2d9d649d492b77adeddea0242ab6b7fe73d18079/packages/next/next-server/server/next-server.ts#L238-L242

**Reply by naseef0** [reply]
ok thank you
